core/src/test/java/org/elasticsearch/search/builder/SearchSourceBuilderTests.java

```diff
@@ -216,16 +217,10 @@ protected final SearchSourceBuilder createSearchSourceBuilder() throws IOExcepti
             }
         }
         if (randomBoolean()) {
-            // NORELEASE make RandomQueryBuilder work outside of the
-            // AbstractQueryTestCase
-            // builder.query(RandomQueryBuilder.createQuery(getRandom()));
-            builder.query(QueryBuilders.termQuery(randomAsciiOfLengthBetween(5, 20), randomAsciiOfLengthBetween(5, 20)));
+            builder.query(RandomQueryBuilder.createQuery(getRandom()));
         }
         if (randomBoolean()) {
-            // NORELEASE make RandomQueryBuilder work outside of the
-            // AbstractQueryTestCase
-            // builder.postFilter(RandomQueryBuilder.createQuery(getRandom()));
-            builder.postFilter(QueryBuilders.termQuery(randomAsciiOfLengthBetween(5, 20), randomAsciiOfLengthBetween(5, 20)));
+            builder.postFilter(RandomQueryBuilder.createQuery(getRandom()));
```

Comment:
I think it's nice to not have to instantiate a new test class to get a random query from it. That said I think I am missing why this change couldn't be made earlier too, I may be overlooking something. Also, isn't the plan to expand how we randomize queries in RandomQueryBuilder? In that case these new static methods don't seem to help a lot cause they would require a huge switch (which we don't have at the moment)?